react-charts/line-chart/step4.md

```diff
@@ -1,71 +1,34 @@
-Now we'll add multiple datasets to the line chart...
+Add tooltips to the line chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
-&lt;ChartLine 
-  data={[
-    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 1 }, 
-    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 2 }, 
-    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 5 }, 
-    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 3 }
-  ]} 
-/&gt;
+&lt;Chart
+  maxDomain={{y: 10}}
+  minDomain={{y: 0}}
+  height={250}
+  width={600}
+&gt;
 </pre>
 

```

Comment:
```suggestion
<strong>Add a `containerComponent` property to that section:</strong>``

Reply:
@gheake2, i changed this file as @christiemolloy asked above.  marking this resolved.